docs/tutorials/PluginDevelopmentGuide.md

```diff
@@ -65,6 +65,8 @@ stackoverflow.com
 
```

Comment:
The grammar is generally not great, when everything else is fixed I'll rework it